domains/Accounts/Resources/Views/verify-email.blade.php

```diff
@@ -0,0 +1,7 @@
+<div>
+    @if($alreadyValidated ?? false)
+        <p>{{ __('Your email was already verified, but thanks anyway for re-confirming it.') }}</p>
+    @else
+        <p>{{ __('Thank you! You can now login at') . ' ' . config('app.name') }}</p>
```

Comment:
The value of the config key `app.name` will be `api.laravel.pt`, and the user won't be able to authenticate themselves through that address. We can either hardcode the `https://laravel.pt` address or use a new environment variable and reference that here.

Reply:
ah I see. Will use an env var to make it flexible.